pkg/api/validate/dynamic/dynamic.go

```diff
@@ -389,6 +389,10 @@ func (dv *dynamic) ValidateSubnets(ctx context.Context, oc *api.OpenShiftCluster
 			}
 		}
 
+		if ss.SubnetPropertiesFormat.ProvisioningState != mgmtnetwork.Succeeded {
```

Comment:
It looks like `ss.SubnetPropertiesFormat` can be nil, can we check for this similar to line 385?

Reply:
Done!